universal-login-wallet/cypress/support/loginAppCommands.ts

```diff
@@ -28,3 +28,11 @@ Cypress.Commands.add('topUpAccount', () => {
 Cypress.Commands.add('goToDashboard', () => {
   cy.get('.modal-success-btn').click();
 });
+
+Cypress.Commands.add('getLocalStorage', (token) => {
+  cy.window().then((window) => JSON.parse(window.localStorage.getItem(token)));
+});
+
+Cypress.Commands.add('checkLocalStorageTokenProp', (token, prop, value) => {
+  cy.getLocalStorage(token).its('kind').should('eq', value);
+});
```

Comment:
Could we please move it to `commands.js` file?

Reply:
Ok.